testworld/start_test.go

```diff
@@ -15,13 +15,13 @@ var isRunningOnCI = len(os.Getenv("TRAVIS")) != 0
 
 // Adjust these based on local testing requirments, please revert for CI server
 var configFile = "configs/local.json"
-var runPOAGeth = !isRunningOnCI
+var runPOAGeth = false
```

Comment:
revert ;)

Reply:
We need better solution than this soon :)

Reply:
we will, its on the way